src/current/molt/molt-fetch.md

```diff
@@ -369,10 +372,71 @@ To cancel replication, enter `ctrl-c` to issue a `SIGTERM` signal. This returns
 `import-only` instructs MOLT Fetch to load the source data in the specified [cloud storage](#cloud-storage) or [local file server](#local-file-server) into the CockroachDB target.
 
 {% include_cached copy-clipboard.html %}
-~~~ 
+~~~ shell
 --mode import-only
 ~~~
 
+#### Fail back to source database
+
+{{site.data.alerts.callout_info}}
+`failback` mode supports MySQL 8.0 and later.
+{{site.data.alerts.end}}
+
+If you encounter issues after moving data to CockroachDB, you can use `failback` to replicate changes on CockroachDB back to the initial source database. In case you need to roll back the migration, this ensures that data is consistent on the source.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--mode failback
+~~~
+
+When running `molt fetch --mode failback`, `--source` is the CockroachDB connection string and `--target` is the connection string of the database from which you migrated. `--table-filter` specifies the tables to watch for change events. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--source 'postgresql://{username}:{password}@{host}:{port}/{database}'
+--target 'mysql://{username}:{password}@{protocol}({host}:{port})/{database}'
+--table-filter 'employees, payments'
+~~~
+
+`failback` mode creates a [CockroachDB changefeed]({% link {{ site.current_cloud_version }}/change-data-capture-overview.md %}) that passes change events from CockroachDB to a [webhook sink]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-sink). However, if there is already a running CockroachDB changefeed with the same webhook sink URL (excluding query parameters) and [watched rows]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}), the existing changefeed is used for `failback`.
```

Comment:
Hmm would it be worth noting that we are also bringing up the webhook sink for them? It's transparent to them since replicator is that webhook sink. I think this wording may be confusing since they may wonder if they need to bring up their own webhook sink

Reply:
Done.